execution/block-partitioner/src/test_utils.rs

```diff
@@ -241,6 +247,12 @@ pub fn verify_partitioner_output(
             {
                 for loc in locs.iter() {
                     let key = loc.clone().into_state_key();
+                    let key_str = CryptoHash::hash(&key).to_hex();
```

Comment:
Thanks to [this](https://github.com/aptos-labs/aptos-core/pull/9883/files#diff-84ffaeeda4c34dea2703a0bbe23ca63b33661faf96eaf67c62489495b10120b4) change from you, this will come free when I merge main.